### PR TITLE
[Api/Tizen] update APIs for next process

### PR DESCRIPTION
**Pull request by jaeyun-jung**
PR for next ACR

1. get the byte size with data index.
2. add new function to set the timeout to wait for the output.

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/capi/include/nnstreamer.h [hunk 1]
```diff
@@ -700,12 +700,17 @@ int ml_tensors_info_set_tensor_dimension (ml_tensors_info_h info, unsigned int i
...
+ * @param[in] info The handle of tensors information.
+ * @param[in] index The index of the tensor.
+ * @param[out] data_size The byte size of tensor data.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
  */
-size_t ml_tensors_info_get_size (const ml_tensors_info_h info);
+int ml_tensors_info_get_tensor_size (ml_tensors_info_h info, unsigned int index, size_t *data_size);
```
Comment (myungjoo) [inline api/capi/include/nnstreamer.h]: What about returning total-size if index=MAX_UINT?

Comment (jaeyun-jung) [reply api/capi/include/nnstreamer.h]: will be updated: if the index is -1, return total byte size of tensors.

**Comment by myungjoo** [comment]
PTAL: @again4you @leemgs

**Comment by again4you** [comment]
LGTM!. If we want to apply the Tizen Release, we need to make a `New` ACR.

**Comment by myungjoo** [comment]
@again4you You need to mark "Approve". I'll proceed assuming that you've simply forgot to click that button.

**Pull request merged**